src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -897,6 +914,45 @@ bool PolicyHandler::UnloadPolicyLibrary() {
   return ret;
 }
 
+#ifdef EXTENDED_PROPRIETARY
+struct SDLAlowedNotification {
+  SDLAlowedNotification(const connection_handler::DeviceHandle& device_id,
+                        PolicyManager* policy_manager,
+                        StateController& state_controller)
+      : device_id_(device_id)
+      , policy_manager_(policy_manager)
+      , state_controller_(state_controller) {}
+
+  void operator()(const ApplicationSharedPtr& app) {
+    if (!policy_manager_) {
+      return;
+    }
+    if (device_id_ == app->device()) {
+      std::string hmi_level;
+      mobile_apis::HMILevel::eType default_mobile_hmi;
+      policy_manager_->GetDefaultHmi(app->policy_app_id(), &hmi_level);
+      if ("BACKGROUND" == hmi_level) {
```

Comment:
@LuxoftAKutsan I found "BACKGROUND" almost 10 times in our project. Isn't it time for a constant? Check others below too.